pkg/storaged/client.js

```diff
@@ -1056,7 +1056,7 @@ function stratis2_start() {
                 };
 
                 client.features.stratis = true;
-                client.stratis_pools = client.stratis_manager.client.proxies("org.storage.stratis2.pool.r1",
+                client.stratis_pools = client.stratis_manager.client.proxies("org.storage.stratis2.pool.r3",
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230525-101055-c79bf30c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)